newsletter/__init__.py

```diff
@@ -0,0 +1,5 @@
+from django.utils.version import get_version
```

Comment:
This is easily unit testable with ``tox`` as well where you don't pin a certain Django version in your ``tox`` requirements, and let newsletter pull in Django itself.